src/tests/test_csv_parser.cpp

```diff
@@ -0,0 +1,95 @@
+#include <cstddef>
```

Comment:
Try to include all standard headers which define names you're using here. These are at least `vector`, `utility`, `string`